src/test/codegen-check/__snapshots__/codegen.test.ts.snap

```diff
@@ -359,6 +389,39 @@ _ %testIfOptimizationNegative(int $a) method_id(87434) {
     
     ;; Handle bounced messages
     if (msg_bounced) { return (); }
+    var op = in_msg~load_opcode_internal();
+    ;; Receive Message1 message
+    if (op == 3110385022) {
+        var $msg'a = in_msg~load_int(257);
+        throw($msg'a);
+        $MainContract$_contract_store(($self'field, $self'value, $self'data, $self'mapping));
+        return ();
+    }
+    
+    ;; Receive Message2 message
+    if (op == 797349837) {
+        var $msg'value = in_msg~load_ref();
+        var $msg'value2 = in_msg~load_ref().begin_parse();
+        throw(($Cell$_fun_depth($msg'value) + $Slice$_fun_depth($msg'value2)));
+        $MainContract$_contract_store(($self'field, $self'value, $self'data, $self'mapping));
+        return ();
+    }
+    
+    ;; Receive Message3 message
+    if (op == 1120645875) {
+        var $msg'val1 = in_msg~load_int(257);
+        var $msg'val2 = in_msg~load_int(257);
+        var $msg'val3 = in_msg~load_int(257);
+        slice sc_1 = in_msg~load_ref().begin_parse();
```

Comment:
What if I name a Tact variable `sc_1`?

Reply:
We add `$` for user variables

![telegram-cloud-photo-size-2-5230933948354395976-x](https://github.com/user-attachments/assets/656bce70-021e-4120-b59a-e08f11e49db1)

![telegram-cloud-photo-size-2-5230933948354395975-y](https://github.com/user-attachments/assets/f1f3b746-296c-4aea-a36a-b4a9a9095828)